dev-requirements.txt

```diff
@@ -6,6 +6,7 @@ Flake8-pyproject~=1.2
 isort~=5.11
 moto~=4.1.3
 pre-commit~=2.21
+pyparsing~=3.0.9
```

Comment:
not sure why I had to add that, but the CI was failing. Locally was working fine for me...somehow moto is not installing it...